src/components/dom.js

```diff
@@ -98,4 +159,126 @@ export default class Dom {
 
     }
 
+    /**
+     * Checks target if it is native input
+     * @param {Element|*} target - HTML element or string
+     * @return {boolean}
+     */
+    static isNativeInput(target) {
+
+        let nativeInputs = [
+            'INPUT',
+            'TEXTAREA'
+        ];
+
+        return target ? nativeInputs.indexOf(target.tagName) !== -1 : false;
+
+    }
+
+    /**
+     * Checks node if it is empty
+     * It must be node without childNodes
```

Comment:
What does it mean? Can I check any Node?

Reply:
It should be noticed somewhere in the description, that I need to use `isEmpty` method instead.